installing/overview/installing-preparing.adoc

```diff
@@ -39,7 +39,7 @@ endif::openshift-origin[]
 
 You can deploy an {product-title} 4 cluster to both on-premise hardware and to cloud hosting services, but all of the machines in a cluster must be in the same data center or cloud hosting service.
 
-If you want to use {product-title} but do not want to manage the cluster yourself, you have several managed service options. If you want a cluster that is fully managed by Red Hat, you can use link:https://www.openshift.com/products/dedicated/[OpenShift Dedicated] or link:https://www.openshift.com/products/online/[OpenShift Online]. You can also use OpenShift as a managed service on Azure, AWS, {ibm-cloud-name}, or Google Cloud. For more information about managed services, see the link:https://www.openshift.com/products[OpenShift Products] page. If you install an {product-title} cluster with a cloud virtual machine as a virtual bare metal, the corresponding cloud-based storage is not supported.
+If you want to use {product-title} but you do not want to manage the cluster yourself, you can choose from several managed service options. If you want a cluster that is fully managed by Red Hat, you can use link:https://www.openshift.com/products/dedicated/[OpenShift Dedicated]. You can also use OpenShift as a managed service on {azure-short}, {aws-short}, {ibm-cloud-name}, or {gcp-full}. For more information about managed services, see the link:https://www.openshift.com/products[OpenShift Products] page. If you install an {product-title} cluster with a cloud virtual machine as a virtual bare metal, the corresponding cloud-based storage is not supported.
```

Comment:
```suggestion
If you want to use {product-title} but you do not want to manage the cluster yourself, you can choose from several managed service options. If you want a cluster that is fully managed by Red Hat, you can use link:https://www.openshift.com/products/dedicated/[OpenShift Dedicated]. You can also use OpenShift as a managed service on {azure-short}, {aws-short}, {ibm-cloud-name}, or {gcp-full}. For more information about managed services, see the link:https://www.openshift.com/products[OpenShift Products] page. If you install an {product-title} cluster with a cloud virtual machine as a virtual bare metal machine, the corresponding cloud-based storage is not supported.
```
out of scope, but wonder if "machine" should be added here

Reply:
Thanks. 'bare metal' seems to be the convention (e.g., https://docs.openshift.com/container-platform/4.17/installing/overview/installing-preparing.html#installing-preparing-security), so I don't think 'machine' is needed here.

Reply:
+1